server/lib/src/entry.rs

```diff
@@ -159,16 +159,22 @@ pub struct EntryReduced {
     uuid: Uuid,
 }
 
+// One day this is going to be Map<Attribute, ValueSet> - @yaleman
+// pub type Eattrs = Map<Attribute, ValueSet>;
```

Comment:
How will that work with custom attributes that users define? Will we need a way for them to make and assign there or wrap these somehow?

Reply:
Yeah, I figure we'll have a `Attribute::CustomAttribute(String)` or something like that?

Probably good to auto-prepend "custom_" on the name when they're created, and maybe hide it when displaying, or something?